intermine_boot/intermine_docker.py

```diff
@@ -0,0 +1,300 @@
+    else:
+        print ('Pulling images...')
+        tomcat_image = client.images.pull('intermine/tomcat:latest')
+        solr_image = client.images.pull('intermine/solr:latest')
+        postgres_image = client.images.pull('intermine/postgres:latest')
+        intermine_builder_image = client.images.pull('intermine/builder:latest')
+
+    print ('Starting containers...')
+    tomcat_container = create_tomcat_container(client, tomcat_image)
+    solr_container = create_solr_container(client, solr_image, env)
+    postgres_container = create_postgres_container(client, postgres_image, env)
+    intermine_builder_container = create_intermine_builder_container(
+        client, intermine_builder_image, env)
+        # # Make sure dockerhub images are up-to-date.
+        # subprocess.run(['docker-compose',
+        #                 '-f', compose_path.name,
+        #                 'pull'],
+        #                check=True,
+        #                cwd=compose_path.parent)
+
+
+    # subprocess.run([*ENV_VARS, *option_vars,
+    #                 'docker-compose',
+    #                 '-f', compose_path.name,
+    #                 'up', '-d'] +
+    #                (['--build', '--force-recreate']
+    #                 if options['build_images'] else []),
+    #                check=True,
+    #                cwd=compose_path.parent)
+    
+    _store_conf(env['data_dir'], options)
+
+
+def down(options, env):
+    compose_path = _get_compose_path(options, env)
+    subprocess.run([*ENV_VARS,
+                    'docker-compose',
+                    '-f', compose_path.name,
+                    'down'],
+                   check=True,
+                   cwd=compose_path.parent)
+
+
+def monitor_builder(options, env):
+    compose_path = _get_compose_path(options, env)
+    # This command will print the logs from intermine_builder and exit
+    # once it finishes building (blocking until then).
+    subprocess.run(['docker-compose',
+                    '-f', compose_path.name,
+                    'logs', '-f', 'intermine_builder'],
+                   check=True,
+                   cwd=compose_path.parent)
+
+
+def create_archives(options, env):
+    compose_path = _get_compose_path(options, env)
+
+    postgres_archive = env['data_dir'] / 'postgres'
+    postgres_data_dir = compose_path.parent / 'data' / 'postgres'
+    shutil.make_archive(postgres_archive, 'zip', root_dir=postgres_data_dir)
+
+    solr_archive = env['data_dir'] / 'solr'
+    solr_data_dir = compose_path.parent / 'data' / 'solr'
+    shutil.make_archive(solr_archive, 'zip', root_dir=solr_data_dir)
+
+    mine_archive = env['data_dir'] / 'biotestmine'
+    mine_data_dir = compose_path.parent / 'data' / 'mine' / 'biotestmine'
+    shutil.make_archive(mine_archive, 'zip', root_dir=mine_data_dir)
+
+
+def create_tomcat_container(client, image):
+    MEM_OPTS = os.environ.get('MEM_OPTS')
+    if MEM_OPTS is None:
+        MEM_OPTS = "-Xmx1g -Xms500m"
```

Comment:
You can just pass a default value to `os.environ.get()` method and avoid two extra lines of code.

Reply:
ok